### PR TITLE
attempt to add spacing, tabs and colors to test output

### PR DESCRIPTION
**Pull request by iamchrissmith**
Goal:
Add some minor formatting improvements to `dapp test` outputs. Specifically, add a couple of line breaks and a visual separator at the beginning of the tests.  Also indent each test result by one tab and add appropriate colors to `PASS`, `FAIL`, and `OOPS`

### CODE REVIEW

File: src/hevm/src/EVM/UnitTest.hs [hunk 1]
```diff
@@ -427,13 +428,13 @@ runUnitTestContract
                      (fromIntegral gasSpent :: Integer)
                in
                  pure
-                   ( "PASS " <> testName <> " (gas: " <> gasText <> ")"
+                   ( "\t\x1b[32mPASS\x1b[0m " <> testName <> " (gas: " <> gasText <> ")"
```
Comment (rainbreak) [inline src/hevm/src/EVM/UnitTest.hs]: would prefer no indents here

**Comment by asymmetric** [comment]
@rainbreak @livnev is this good to go?

**Comment by iamchrissmith** [comment]
@asymmetric, @rainbreak said he was going to integrate it into some related work he was doing, so this might be closable
(ref: https://dapphub.chat/channel/dev?msg=PbErzCfPagCXHC8wd)

**Comment by rainbreak** [comment]
Yeah, I've got a local branch with some more changes.

**Comment by iamchrissmith** [comment]
@rainbreak should I close this at this point?

**Comment by rainbreak** [comment]
@iamchrissmith yes, this was superseded and master now has colours :)

**Pull request closed without merging**